src/main/java/uk/gov/companieshouse/pscstatementdataapi/services/PscStatementService.java

```diff
@@ -127,28 +151,37 @@ private Created getCreatedFromCurrentRecord(String companyNumber,String statemen
   }
 
   private StatementList createStatementList(List < PscStatementDocument > statementDocuments,
-                                            int startIndex, int itemsPerPage, Optional < MetricsApi > metrics, String companyNumber) {
+                                            int startIndex, int itemsPerPage, MetricsApi companyMetrics, String companyNumber, boolean registerView) {
+
     StatementList statementList = new StatementList();
     StatementLinksType links = new StatementLinksType();
     links.setSelf(String.format("/company/%s/persons-with-significant-control-statements", companyNumber));
+
     List < Statement > statements = statementDocuments.stream().map(PscStatementDocument::getData).collect(Collectors.toList());
+
+    if (registerView) {
+      Long withdrawnCount = statements.stream()
+              .filter(statement -> statement.getCeasedOn() != null).count();
+
+        if (withdrawnCount > 0) {
+          statementList.setCeasedCount(withdrawnCount.intValue());
+          statementList.setTotalResults(companyMetrics.getCounts().getPersonsWithSignificantControl().getActiveStatementsCount()
+                  + statementList.getCeasedCount());
+        } else {
+          statementList.setCeasedCount(companyMetrics.getCounts().getPersonsWithSignificantControl().getWithdrawnStatementsCount());
```

Comment:
I'm not sure this if check is right. We're defaulting back to metrics inside registerView which means we could show statements that were removed prior to the registerMovedOn date